src/tools/cgget.c

```diff
@@ -746,6 +759,14 @@ int main(int argc, char *argv[])
 	if (ret)
 		goto err;
 
+	if (mode & MODE_SYSTEMD_DELEGATE) {
+		ret = cgroup_get_default_slice_name(CGCONFIG_CONF_FILE);
```

Comment:
What if the user is utilizing the cgconfigconfdir, `/etc/cgconfig.d`?

Reply:
@drakenclimber Thanks for all the corner cases and functionality, I had missed. I will re-work them and share the changes sooner.